pinot-core/src/main/java/org/apache/pinot/core/common/BlockValSet.java

```diff
@@ -29,6 +29,13 @@
  */
 public interface BlockValSet {
 
+  /**
+   * Returns the number of entries for a single-valued column.
+   *
+   * @return number of single-valued entries
+   */
+  int getNumSVEntries();
```

Comment:
I don't think we need this API. If it is really required, let's rename it to `getNumDocs()`